### DISCUSSION TITLE
How can I insert more rows into a  table after an image pasted in a table?

### DISCUSSION THREAD

**Discussion by gerroon** [TriliumNext/Trilium#2721] [Q&A] [answered]
Hi

I create a one or two row table and paste an image. So far so good, but after pasting an image into a table I can not get the table menu ever again, to add/remove rows and columns. How do I get the menu back in that situation?

**Comment by XXXJetfireXXX** [comment, marked as answer]
A temporary workaround would be to click on an image once, press `->` and then type anything you want (`space` works too). This'll get the menu to pop up again.
